src/test/statistics/statistics_objects/generic_histogram_test.cpp

```diff
@@ -768,8 +768,7 @@ TEST_F(GenericHistogramTest, SplitAtEmptyBinBounds) {
   const auto expected_minima = std::vector<int32_t>{};
   const auto expected_maxima = std::vector<int32_t>{};
   const auto expected_heights = std::vector<HistogramCountType>{};
-  const auto expected_distinct_counts =
-      std::vector<HistogramCountType>{};
+  const auto expected_distinct_counts = std::vector<HistogramCountType>{};
```

Comment:
I guess these changes come from a clang format version mismatch?

Reply:
Probably. I ran the formatter on the sidon server: clang-format version 7.0.1-8 (tags/RELEASE_701/final)